src/biogeochem/DUSTMod.F90

```diff
@@ -181,6 +315,14 @@ subroutine InitCold(this, bounds)
        end if
     end do
 
+      ! Caulculate Drag Partition factor
+      if ( this%prigent_roughness_stream%useStreams() )then !if usestreams == true, and it should be always true
+         call this%CalcDragPartition( bounds, this%prigent_roughness_stream )
+      else
+
+         call endrun( "ERROR:: Drag partitioning MUST now use a streams file of aeolian roughness length to calculate, it can no longer read from the fsurdat file" )
```

Comment:
I'll update the error message so that it just says that Prigent streams are required. Actually for Paleo we need to allow turning it off. So perhaps we should have a warning rather than an abort? Or maybe it also checks something else like the orbital year to see if it should be allowed to turn it off.